webknossos/webknossos/dataset/_array.py

```diff
@@ -504,26 +505,30 @@ def info(self) -> ArrayInfo:
 
         zarray = self._zarray
         if isinstance(zarray, Array):
-            if len(zarray.codecs) == 1 and isinstance(zarray.codecs[0], ShardingCodec):
-                sharding_codec = zarray.codecs[0]
+            if len(zarray.codec_pipeline.codecs) == 1 and isinstance(
+                zarray.codec_pipeline.codecs[0], ShardingCodec
+            ):
+                sharding_codec = zarray.codec_pipeline.codecs[0]
+                shard_shape = zarray.metadata.chunk_grid.configuration.chunk_shape
+                chunk_shape = zarray.metadata.chunk_grid.configuration.chunk_shape
```

Comment:
`shard_shape` and `chunk_shape` are both initialized with the same value.

Reply:
good catch!